releases/fluentd-splunk-logs.yaml

```diff
@@ -0,0 +1,52 @@
+releases:
+
+#######################################################################################
+## Fluentd to Splunk logs                                                            ##
+## Forward logs to Splunk with fluentd                                               ##
+#######################################################################################
+
+#
+# References:
+#   - https://github.com/splunk/splunk-connect-for-kubernetes/blob/master/helm-chart/splunk-kubernetes-logging/values.yaml
+#
+
+- name: "fluentd-splunk-logs"
+  namespace: "kube-system"
+  labels:
+    chart: "splunk-kubernetes-logging"
+    repo: "splunk"
+    component: "splunk"
+    namespace: "kube-system"
+    default: "false"
+  chart: "https://github.com/splunk/splunk-connect-for-kubernetes/releases/download/v1.0.1/splunk-kubernetes-logging-1.0.1.tgz"
```

Comment:
Ohhhh this! that's nice. So now it's not really necessary to have a chart registry.